<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ARCUAS UAV CTRL </string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>NET:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lab_NetStatus">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 0, 0);</string>
            </property>
            <property name="text">
             <string>DISCONNECT</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="editLog"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_Winch" native="true">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>271</width>
         <height>381</height>
        </rect>
       </property>
       <property name="title">
        <string>WinchTest</string>
       </property>
       <widget class="QPushButton" name="BtnWinchTest">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>120</y>
          <width>51</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
       <widget class="QPushButton" name="BtnWinch_Reset">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>200</y>
          <width>71</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>38</y>
          <width>70</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>CurrentPos:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_WinchPos">
        <property name="geometry">
         <rect>
          <x>94</x>
          <y>117</y>
          <width>91</width>
          <height>32</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-100000</number>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_AutoTest">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>330</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 255, 0);</string>
        </property>
        <property name="text">
         <string>AutoTest</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_CurrentWinchPos">
        <property name="geometry">
         <rect>
          <x>116</x>
          <y>28</y>
          <width>131</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 255, 0);
font: 75 18pt &quot;PibotoLt&quot;;
background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>123</y>
          <width>61</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>SetPos:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_WinchSpeed">
        <property name="geometry">
         <rect>
          <x>94</x>
          <y>150</y>
          <width>91</width>
          <height>32</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-100000</number>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>-140</x>
          <y>90</y>
          <width>61</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>SetPos:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>154</y>
          <width>61</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>speed:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="BtnWinch_Delivery">
        <property name="geometry">
         <rect>
          <x>106</x>
          <y>200</y>
          <width>71</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Delivery</string>
        </property>
       </widget>
       <widget class="QPushButton" name="BtnWinch_Retract">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>200</y>
          <width>71</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Retract</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Btn_Motor_dump_off">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>290</y>
          <width>99</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>stop</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Btn_Motor_data_dump">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>290</y>
          <width>99</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>start data send</string>
        </property>
       </widget>
       <widget class="QPushButton" name="BtnWinch_LOAD">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>250</y>
          <width>70</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>LOAD</string>
        </property>
       </widget>
       <widget class="QPushButton" name="BtnWinch_LOAD_UP">
        <property name="geometry">
         <rect>
          <x>107</x>
          <y>250</y>
          <width>70</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>LOAD_UP</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>31</x>
          <y>82</y>
          <width>70</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Torque:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_CurrentWinchTorque">
        <property name="geometry">
         <rect>
          <x>116</x>
          <y>72</y>
          <width>130</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 255, 0);
font: 75 18pt &quot;PibotoLt&quot;;
background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>540</y>
         <width>271</width>
         <height>201</height>
        </rect>
       </property>
       <property name="title">
        <string>SC15(20~1000)</string>
       </property>
       <widget class="QPushButton" name="Btn_Motor_CCW">
        <property name="geometry">
         <rect>
          <x>142</x>
          <y>110</y>
          <width>99</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>CCW</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Btn_Motor_CW">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>99</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>CW</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_CurrentSCPos">
        <property name="geometry">
         <rect>
          <x>116</x>
          <y>30</y>
          <width>131</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 255, 0);
font: 75 18pt &quot;PibotoLt&quot;;
background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>CurrentPos:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="BtnSC15_SetPos">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>74</y>
          <width>51</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_SC15Pos">
        <property name="geometry">
         <rect>
          <x>94</x>
          <y>71</y>
          <width>91</width>
          <height>32</height>
         </rect>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>77</y>
          <width>61</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>SetPos:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="SC15_AutoTest">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>150</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 255, 0);</string>
        </property>
        <property name="text">
         <string>AutoTest</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="B_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>400</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
border:3px groove gray;
border-style:outset;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="B_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>400</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
border:3px groove gray;
border-style:outset;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>430</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>D18</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>430</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>D22</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>98</x>
         <y>430</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>SCL</string>
       </property>
      </widget>
      <widget class="QLabel" name="SCL">
       <property name="geometry">
        <rect>
         <x>98</x>
         <y>400</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
border:3px groove gray;
border-style:outset;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="SDA">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>400</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
border:3px groove gray;
border-style:outset;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>430</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>SDA</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
